src/pages/Home/Banner/index.tsx

```diff
@@ -15,9 +16,9 @@ const _Banner: FC<{ latestBlock?: State.Block }> = ({ latestBlock }) => {
     const container = ref.current
     if (!container) return
     try {
-      const r = createBannerRender(container)
-      setRender(r)
-      return () => r.destroy()
+      let r: BannerRender
+      createPioneerRender(container, setRender, () => createBannerRender(container))
```

Comment:
The code design at this point is a bit strange. `createPioneerRender` shouldn't directly require `setRender` and `next` parameters and process them, as it doesn't match its name. Additionally, it's unnecessary to provide a complete `createPioneerRender` that includes the rendering loop. Only a `renderTinyTestScene(container)` or `renderTinyRenderTest(container)` function is needed.

A more reasonable design might be as follows:
1. Measure time externally, for example:
```
const start = Date.now()
renderTinyTestScene(container)
const time = Date.now() - start
if (time < 16) {
  const r = createBannerRender(container)
  setRender(r)
  return () => r.destroy()
}
```
2. Directly return the time, for example:
```
const time = renderTinyTestScene(container)
if (time < 16) {
  const r = createBannerRender(container)
  setRender(r)
  return () => r.destroy()
}
```

Reply:
refactor to render new banner according to render time of tiny scene